app/src/features/mocks/screens/mocksList/components/MocksList/MocksList.tsx

```diff
@@ -27,6 +28,13 @@ import {
   UpdateMockCollectionModal,
 } from "features/mocks/modals";
 import "./mocksList.scss";
+import { message } from "antd";
```

Comment:
Why are we using message when we have out internal interface for toast too. We should use that only